src/plugins/ganttChart/ganttChart.js

```diff
@@ -405,17 +428,26 @@ class GanttChart extends BasePlugin {
    * Get the week column range.
    *
    * @private
-   * @param {Object} monthObject The month object.
-   * @param {Number} monthNumber Index of the month.
-   * @param {Number} headerIndex Index of the header.
-   * @param {Boolean} mixedMonth `true` if the header is the single header of a mixed month.
+   * @param {Object} options The options object.
+   * @param {Object} options.monthObject The month object.
+   * @param {Number} options.monthNumber Index of the month.
+   * @param {Number} options.headerIndex Index of the header.
+   * @param {Boolean} options.areDaysBeforeFullWeeksVisible `true` if the days before full weeks are to be visible.
+   * @param {Boolean} options.areDaysAfterFullWeeksVisible `true` if the days after full weeks are to be visible.
+   * @param {Number} options.headerCount Number of headers to be generated for the provided month.
+   * @param {Boolean} [options.mixedMonth] `true` if the header is the single header of a mixed month.
```

Comment:
Please change from `[options.mixedMonth]` to `[options.mixedMonth=false]` and check that JSDoc by building the documentation.